### ISSUE TITLE
Create accessibility training program for CivicActions team members

### ISSUE THREAD

**Issue by lukefretwell** [CivicActions/accessibility#3]
@elizabethraley, @mgifford and I are talking about developing a high-level accessibility training (30-60 minutes) for CivicActions team members. We'd like to test on groups of current employees and then do monthly for new employees.

Who's the best person to talk with about this?

## Audience

CivicActions team members (and eventually clients)

## Objective

To educate CivicActions team members on web accessibility and the importance of this in our work with government.

### Key results

* 100% of CivicActions team members attend digital accessibility training.
* All CivicActions team members score 90% or higher on a digital accessibility quiz.
* 100% of CivicActions team members are using appropriate accessibility tools.

**Comment by mgifford** [comment]
This will be really important for developing a common approach to implementing accessibility for our clients, but also creating a common culture where disability can be embraced as an important part of the human experience. There is considerable stigma associated with disability, and many disabilities are either episodic or invisible.

**Comment by mgifford** [comment]
Adding link to draft onboarding workshop - https://github.com/CivicActions/accessibility/blob/main/Onboarding.md
Main twitter thread - https://twitter.com/mgifford/status/1348990752412602371

**Comment by dmundra** [comment]
@mgifford, should we follow security's lead and create a quiz to test folks on accessibility basics and the material in the training/onboarding document?

Current security onboarding steps:
* Read the following:
  * https://handbook.civicactions.com/en/latest/010-welcome-to-civicactions/training/security-training/
  * https://docs.google.com/presentation/d/1BM4w54QVMufBZyfUZRhSDKhU94Ap5v_TTmFpOtylTOc/edit#slide=id.gb3711abec_1_2
  * https://handbook.civicactions.com/en/latest/100-security/incident-response-plan/
  * https://handbook.civicactions.com/en/latest/100-security/contingency-plan/
* Take quiz: https://docs.google.com/forms/d/e/1FAIpQLScv6acnINdgaENE_jucXq1LrrfDb6-lmCmNTaw6yjPxD9o-DA/viewform

We could do the same and also include the AHC presentation #45 in the reading material. Thoughts?

**Comment by mgifford** [comment]
Thanks @dmundra I'd love to have you working with me on building this out. Kylea is looking to update how we onboard folks.  

And perfect to build on the security teams existing work. There is lots of similarities. 

This points to us needing content in the CA Handbook:
- [ ] A central Accessibility for Everyone page 
- [ ] Accessibility policies
- [ ] CivicActions approved courses / certifications
- [ ] Reporting barriers

Also
- [ ] Common presentation to watch or walk through - #45
- [ ] A quiz where we want everyone to take (and to report to the accessibility channel) - do we have an 80% target again?

I don't know if we need the incident response or contingency plan.

**Comment by dmundra** [comment]
@mgifford is the third video on https://github.com/CivicActions/accessibility/blob/main/Onboarding.md this one: https://www.youtube.com/watch?v=vK1tlLOavvM?

**Comment by mgifford** [comment]
Yes..  I think that's a copy/paste error.  I just corrected it. Thanks @dmundra

**Comment by lukefretwell** [comment]
+ monthly new employee coffee/tea with accessibility team

**Comment by dmundra** [comment]
A CA AHC by @mgifford https://civicactions.zoom.us/rec/play/I9vZM2ijkeSK0dX4D2bwQDSSYVKVeXbjGCVHwBAR04YIjAPKjci69ekbc-S0EfC02IEaawRBuYOGmrPu.0tht7kQVE1Z5vtT0?continueMode=true

**Comment by mgifford** [comment]
We should reference this video https://www.youtube.com/watch?v=ITUDiTgAZY0

**Comment by mgifford** [comment]
We're building something out for new staff onboarding based on the Onboarding.md file in this repo.  

We'll need a [presentations slide deck on accessibility onboarding](https://docs.google.com/presentation/d/1cSfwSOsBZV95eIiv-DPe5bOLV6_7NSYv_O9AOlDbyvw/edit#slide=id.gd852250066_8_74), so I've just started up this one as one we can build on.

**Comment by VanessaLuxen** [comment]
Added one comment mentioning plain language on slide 4: https://docs.google.com/presentation/d/1cSfwSOsBZV95eIiv-DPe5bOLV6_7NSYv_O9AOlDbyvw/edit#slide=id.gd85224ffcd_0_5

**Comment by mgifford** [comment]
Thanks for adding a comment there about plain language. I edited it a bit to make it shorter. Let me know what you think..
